packages/react/src/views/QuoteMessage/QuoteMessage.js

```diff
@@ -53,62 +54,13 @@ const QuoteMessage = ({ className = '', style = {}, message }) => {
         <Box>{format(new Date(message.ts), 'h:mm a')}</Box>
       </Box>
       <Box css={styles.message}>
-        {message.file ? (
```

Comment:
Any specific reason, you are removing these ?

Reply:
sorry, I mistakenly removed the code